frontend/src/scenes/billing/billingProductLogic.ts

```diff
@@ -202,14 +202,17 @@ export const billingProductLogic = kea<billingProductLogicType>([
         ],
     })),
     listeners(({ actions, values, props }) => ({
-        loadBillingSuccess: actions.billingLoaded,
```

Comment:
this listener never gets triggered so I moved the billingloaded call into afterMount below.